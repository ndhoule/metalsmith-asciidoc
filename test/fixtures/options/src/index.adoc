= Asciidoc Roolz

OMG it's _WAY_ better than *Markdown*, or any of those other markup languages.
