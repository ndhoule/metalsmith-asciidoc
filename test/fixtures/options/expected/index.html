<h1>Asciidoc Roolz</h1>
<div class="paragraph">
<p>OMG it&#8217;s <em>WAY</em> better than <strong>Markdown</strong>, or any of those other markup languages.</p>
</div>